docs/development/new-kubernetes-version.md

```diff
@@ -108,7 +108,6 @@ After the PR in `gardener/gardener` for the support of the new version has been
 
 > Actually, you can already start even if the PR is not yet merged and use the branch of your fork.
 
-- Revendor the `github.com/gardener/gardener` dependency in the extension and update the `README.md`.
```

Comment:
Maybe instead of removing this line, we can rephrase it as,
```
- Update the `github.com/gardener/gardener` dependency in the extension and update the `README.md`.
```
since we still need to update gardener dependency and also `README.md` in the extensions. For extensions still using vendoring, it could mean revendor.

Reply:
Fixed.